### ISSUE TITLE
Trainging data of Video Alignment Stage

### ISSUE THREAD

**Issue by QiushiYang** [KwaiVGI/VideoAlign#8]
Great work! 

I have two issues:

(1) It seems there is no description on the training data for video alignment process in the paper. What manner is the training dataset curated and how is the preference for input video pairs determined? How many video pairs are in the training set?

(2) In my understanding, the perferred video pairs during the video alignemnt process are determined by the reward model online, right?

**Comment by yifan123** [comment]
Thank you for your interest in our work!

For issue (1):​​
As described in Section 5.2 of the paper, for the training dataset, we follow the methodology of DPO (Rafailov et al., 2024). Specifically, we use ​​VideoReward​​ as the ground-truth reward model to simulate human feedback and synthetically relabel our training dataset. This ensures that T2V models trained on this relabeled dataset can be fairly and reliably evaluated using VideoReward. The preference for input video pairs is determined ​​offline​​ by the VideoReward model, which ranks pairs based on predicted rewards. The size of the dataset used for DPO training matches the scale of the dataset used to train the VideoReward model.

​​For issue (2):​​
You are correct that the preferred video pairs during alignment are determined by the reward model. However, this process is ​​not conducted online​​. Instead, we perform a single offline DPO step using pre-computed preferences from VideoReward. Importantly, the reward model (VideoReward) serves a dual purpose: it is not only used for DPO training but also acts as a foundational component for ​​more advanced post-training methods in future work.

Let me know if further clarification is needed! 😊

**Comment by QiushiYang** [comment]
Thank you for your reply!

I see the process on video alignment, while I still have two confuses:

(1) The ref paper you mentioned (Rafailov, R., et. al., Direct preference optimization: Your language model is secretly a reward model, DPO) focuses on LLM task and does not contain video data. Do you mean the recent paper "Hiroki Furuta, et. al., Improving Dynamic Object Interactions in Text-to-Video Generation with AI Feedback"? If it is, this paper claims that they selects text prompts from someting-something-v2 dataset and generate multiple videos with those text prompts as the preference videos for DPO training. Do you utilize the same manner to perform video alignment?

(2) I notice you mentioned "VideoReward model can also servers as a  foundational component for ​​more advanced post-training methods", what do you mean? My understanding is that the reward model is mainly used as a human perference predictor to guide RL training, or directly as an video quality evaluation manner, what other uses does it have?

**Comment by yifan123** [comment]
Thanks again for your interest in our work!

(1) We use videoreward to relabel human preference data for training DPO, an approach commonly seen in LLM alignment known as synthetic experiments. This treats videoreward as the gold reward, ensuring more accurate evaluation of different algorithms' optimization capabilities. For reference, see the Synthetic feedback section in MODPO (Beyond One-Preference-Fits-All Alignment: Multi-Objective Direct Preference Optimization).
(2) Videoreward can be used in online RL algorithms such as PPO as a reward model or as a reward for inference methods.
